### PR TITLE
feat:result page with score

### PR DESCRIPTION
**Pull request by mariyasada**
add result page to quiz app.
adding media query.
result

### CODE REVIEW

**Review by fahd23 — APPROVED** [review]
Good to merge

File: Htmlfiles/result-page.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      rel="stylesheet"
+      href="https://iqra-ui.netlify.app/stylesheets/main.css"
+    />
+    <link rel="stylesheet" href="/main.css" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
```
Comment (Kothalkarpriya) [inline Htmlfiles/result-page.html:17]: I didn't understand why you /> comes on next line every second time. Please Look into it , otherwise Good to merge

**Review by Kothalkarpriya — APPROVED** [review]
Look into my comment, Good to merge after it

File: Htmlfiles/result-page.html (hunk 1 above)
Comment (mariyasada) [reply Htmlfiles/result-page.html:17]: it is because of Prettier formatter. when you save your code prettier sets code like this

**Pull request merged**